api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/ClaimantDTO.java

```diff
@@ -75,4 +76,8 @@
     @ApiModelProperty(notes = "The claimant's email address. e.g. test@email.com")
     private String emailAddress;
 
+    @JsonProperty("childrenDob")
+    @ApiModelProperty(notes = "The dates of birth of the claimant's children (if they have any)")
+    private List<LocalDate> childrenDob;
```

Comment:
Should have ListOfDatesInPast annotation.

Reply:
Yep, good spot. This does show that we don't have unit tests for the ClaimDTO, I'll add in another PR.